examples/step-79/doc/intro.dox

```diff
@@ -0,0 +1,597 @@
+
+The optimality conditions above are, in addition to being convoluted, of a kind
+that is not easy to solve: They are generally nonlinear, and some of the
+relationships are also inequalities. We will address the nonlinearity using a
+Newton method to compute search directions, and come back to how to deal with
+the inequalities below when talking about step length procedures.
+
+Newton's method applied to the equations above results in the following system.
+Here, variational derivatives with respect to the $\{\cdot\}$ variable are taken
+ in the $c_{\{\cdot\}}$ direction. This gives
+
+<ol>
+<li> Stationarity - these equations ensure we are at a critical point of the
+objective function when constrained
+
+Equation 0
+@f{multiline}{
+  \int_\Omega-d_\rho c_{y_2} + p(p-1) \rho^{p-2} d_\rho c_\rho [\lambda \nabla
+  \cdot \textbf{y}_1 \nabla \cdot \textbf{u} +  \mu  \varepsilon(\textbf{u})
+  \varepsilon(\textbf{y}_1)] @f]@f[+ p \rho^{p-1} d_\rho[\lambda \nabla \cdot
+  \textbf{c}_{\textbf{y}_1} \nabla \cdot \textbf{u} +   \mu  \varepsilon
+  (\textbf{u}) \varepsilon(\textbf{c}_{\textbf{y}_1})]  +  p \rho^{p-1} d_\rho
+  [\lambda \nabla \cdot {\textbf{y}_1} \nabla \cdot \textbf{c}_\textbf{u} +
+  \mu  \varepsilon(\textbf{c}_\textbf{u}) \varepsilon(\textbf{y}_1)]
+ \\=-\int_\Omega -d_\rho z_1 + d_\rho z_2 - d_\rho y_2 + p\rho^{p-1}d_\rho
+[\lambda \nabla \cdot \textbf{y}_1 \nabla \cdot \textbf{u} + \mu \varepsilon
+(\textbf{u})\varepsilon(\textbf{y}_1)]
+@f}
+
+Equation 1
+@f{multiline}{
+  \int_\Omega p \rho^{p-1} c_\rho [\lambda \nabla \cdot {\textbf{y}_1} \nabla
+  \cdot \textbf{d}_\textbf{u} +  \mu  \varepsilon(\textbf{d}_\textbf{u})
+  \varepsilon(\textbf{y})] + \rho^{p} [\lambda \nabla \cdot
+  \textbf{c}_{\textbf{y}_1} \nabla \cdot \textbf{d}_\textbf{u} +  \mu
+  \varepsilon(\textbf{d}_\textbf{u})\varepsilon(\textbf{c}_{\textbf{y}_1})]
+\\
+=-\int_\Gamma \textbf{d}_\textbf{u} \cdot \textbf{t} -\int_\Omega \rho^{p}
+[\lambda \nabla \cdot \textbf{y} \nabla \cdot \textbf{d}_\textbf{u} + \mu
+\varepsilon(d_\textbf{u})\varepsilon(\textbf{y}_1)]
+@f}
+Equation 2
+@f[
+  \int_\Omega  - d_\sigma c_{z_1} +d_\sigma c_{z_2}  + H(d_\sigma)c_{y_2} =
+  -\int_\Omega -d_\sigma z_1 + d_\sigma z_2 + H(d_\sigma)y_2
+@f]
+</li>
+<li> Primal Feasibility - these equations ensure the equality constraints
+are met.
+
+Equation 3
+@f{multiline}{\int_\Omega p \rho^{p-1} c_p[\lambda \nabla \cdot
+\textbf{d}_{\textbf{y}_1} \nabla \cdot \textbf{u} +  \mu
+\varepsilon(\textbf{u}) \varepsilon(\textbf{d}_{\textbf{y}_1})] +
+\rho^{p}[\lambda \nabla \cdot \textbf{d}_{\textbf{y}_1} \nabla \cdot
+\textbf{c}_\textbf{u} +  \mu  \varepsilon(\textbf{c}_\textbf{u})
+\varepsilon(\textbf{d}_{\textbf{y}_1})]
+\\
+ =-\int_\Omega \rho^{p}[\lambda \nabla \cdot \textbf{d}_{\textbf{y}_1} \nabla
+ \cdot \textbf{u} + \mu  \varepsilon(\textbf{u}) \varepsilon
+ (\textbf{d}_{\textbf{y}_1})]  + \int_\Gamma  \textbf{d}_{\textbf{y}_1}
+ \cdot \textbf{t}
+@f}
+
+    Equation 4
```

Comment:
Hmm... perhaps you could tidy this up (the line indentation for the "Equation" headers, that is).